src/pages/home/Navbar.jsx

```diff
@@ -8,28 +9,21 @@ const NavbarItem = ({title, link, ...props}) => (
 	</a>
 );
 
-const Navbar = ({children, headerRef}) => {
+const Navbar = ({children, clientHeight}) => {
 	const [scrolled, setScrolled] = useState(false);
 
+	const scrollFunc = useCallback(
+		({
+			target: {
+				defaultView: {pageYOffset}
+			}
+		}) => setScrolled(pageYOffset >= clientHeight),
+		[clientHeight]
+	);
+
 	// This is an imperitave solution for this,

```

Comment:
Are these comments still relevant? (see this and line below.)

Reply:
Yeah this is still pretty imperative since it's using an exact value extracted from the rendered DOM node to calculate the component's behavior. Although it's pretty obvious how imperative this is, so the comment doesn't really add anything useful here.